### PR TITLE
Fixes issues with responsiveness

### LINKED ISSUE #180: map - hard to click on the lower regions
make map smaller or take it up a bit

### PR DESCRIPTION
**Pull request by sofiapoh**
closes #180 #171 #169 #167

### CODE REVIEW

File: public/css/jqueryfullPage.css [hunk 1]
```diff
@@ -174,7 +174,7 @@ html.fp-enabled,
     height: 4px;
     width: 4px;
     border: 0;
-    background: #333;
+    background: #666666;
```
Comment (gabrielperales) [inline public/css/jqueryfullPage.css:177]: Evil color 😈

File: public/css/jqueryfullPage.css [hunk 2]
```diff
@@ -184,15 +184,16 @@ html.fp-enabled,
     transition: all 0.1s ease-in-out;
 }
 #fp-nav ul li:hover a span,
-.fp-slidesNav ul li:hover a span{
+.fp-slidesNav ul li:hover a span{
```
Comment (gabrielperales) [inline public/css/jqueryfullPage.css:187]: This rules are over nested, are you sure this won't work?

``` css
.fp-slidesNav :hover span {
// whatever
}
```

and better: 

``` css
.listItem:hover .anotherClassForMySpan {
\\ your rules
}
```

**Comment by gabrielperales** [comment]
Well done!! ✨  🎉

File: public/css/jqueryfullPage.css (hunk 2 above)
Comment (sofiapoh) [reply public/css/jqueryfullPage.css:187]: It's the fullpage library css, i just had to change the colour of the navigation buttons but I agree!

**Pull request merged**